nominees/covid-19-api.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Covid-19 API",
+  "description": "This API can be utilised by all applications that need to write/retrieve Ethiopia Covid19 data. E.g. covid19.et project will use it to retrieve data for dashboard and write data",
+  "website": "https://github.com/africa-covid-19-response-toolkit/api",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/africa-covid-19-response-toolkit/api/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
```

Comment:
@rachita19082 in as much as possible, it would be advantageous to add either `evidenceText` or `evidenceURL` for each SDG entry. At least one of them is required as per the [documentation](https://github.com/unicef/publicgoods-candidates/blob/master/CONTRIBUTING.md). Thanks.

Reply:
@nathanbaleeta Yes understood, I have made the required changes. I will keep this in mind.

Reply:
Resolved in commit [d66e6da](https://github.com/unicef/publicgoods-candidates/pull/381/commits/d66e6da45c2ffe3a4b883ec9255de647fa04253d) and [d240a2a](https://github.com/unicef/publicgoods-candidates/pull/381/commits/d240a2afe888eda864c3e39efbc93a4d2648844e)